### PR TITLE
Allow setting multiple properties on a port

### LINKED ISSUE #63: creating a patch port
The set argument seems to allow setting only one property on a port. I need to create a patch port that requires two properties. Can this be accomplished as is somehow? I have not been able to find a way. Maybe set could accept an array of strings?

Example command:
```
ovs-vsctl add-port physical_br physical_br_patch -- set interface physical_br_patch type=patch -- set interface physical_br_patch option:peer=ovs_br_patch
```

### PR DESCRIPTION
**Pull request by NilashishC**
Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

##### SUMMARY
- Fixes #63 

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
openvswitch_port.py

### CODE REVIEW

**Comment by NilashishC** [comment]
recheck

File: tests/unit/modules/network/ovs/test_openvswitch_port.py [hunk 1]
```diff
@@ -261,3 +261,28 @@ def test_openvswitch_port_present_runs_set_mode(self):
...
+    def test_openvswitch_port_present_runs_set_mode_multiple(self):
+        set_module_args(
+            dict(
+                state="present",
+                bridge="test-br",
+                port="eth2",
+                tag=10,
+                external_ids={"foo": "bar"},
+                set=[
+                    "port eth2 other_config:stp-path-cost=10",
```
Comment (Qalthos) [inline tests/unit/modules/network/ovs/test_openvswitch_port.py:274]: If this is how this works, that's fine, but this seems oddly redundant. Is there anything stopping someone from setting properties of unrelated ports?

I would have expected something closer to

```
state="present",
bridge="test-br",
port="eth2",
tag=10,
external_ids={"foo": "bar"},
set_properties=[
    "other_config:stp-path-cost=10",
    "type=patch",
]
```

or even something like this, though even here the properties are a little hard to name

```

set_properties={
    "other_config:stp-path-cost": 10,
    "type": "patch",
}
```

I'm not expecting a change to this PR, and maybe there's some reason you want to set arbitrary properties, just want to see if this makes sense to deprecate `set` for in the future.

**Review by Qalthos — APPROVED** [review]

File: tests/unit/modules/network/ovs/test_openvswitch_port.py (hunk 1 above)
Comment (NilashishC) [reply tests/unit/modules/network/ovs/test_openvswitch_port.py:274]: @Qalthos I totally agree with you here. The way the `set` key has been designed from the beginning isn't ideal. We should deprecate and replace it with a key that only accepts structured values.

Comment (Qalthos) [reply tests/unit/modules/network/ovs/test_openvswitch_port.py:274]: :+1:

**Pull request merged**